pkg/service/repair/service.go

```diff
@@ -149,37 +149,25 @@ func (s *Service) GetTarget(ctx context.Context, clusterID uuid.UUID, properties
 		return t, errors.Wrap(ErrEmptyRepair, err.Error())
 	}
 
-	t.plan, err = newPlan(ctx, t, client)
+	p, err := newPlan(ctx, t, client)
 	if err != nil {
 		return t, errors.Wrap(err, "create repair plan")
 	}
 	// Sort plan
-	t.plan.SizeSort()
-	t.plan.PrioritySort(NewInternalTablePreference())
+	p.SizeSort()
+	p.PrioritySort(NewInternalTablePreference())
```

Comment:
This is just for me to have better understanding.

Why do you sort the plan here, on the `GetTarget`, but not on the actual repair `service.Repair` ?

Reply:
So that the `--dry-run` shows an actual repair order.